munge_sql.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * This script takes the munged JSON files and puts them into a SQLite database.
+ */
+const sqlite3 = require('sqlite3').verbose();
```

Comment:
What does verbose do? Do we want it always on or only when debugging? FYI, `RUNNER_DEBUG=1` env var indicates that a workflow run is being rerun with debugging logging enabled (if that's useful).

Reply:
We'll get more detailed stack traces on exceptions, which we'll be helpful even without debug logging enabled.
https://github.com/TryGhost/node-sqlite3/wiki/API#sqlite3verbose